wrappers/openvino/face/CMakeLists.txt

```diff
@@ -25,6 +25,10 @@ if(OPENVINO2019)
 	target_compile_definitions(rs-face-vino PRIVATE OPENVINO2019)
 endif()
 
+if(OPENVINO_NGRAPH)
```

Comment:
elseif()?

Reply:
define ngraph compilation definition only when it's available.